mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/mapping/definition/programmatic/TypeMappingStep.java

```diff
@@ -23,11 +26,29 @@
 	TypeMappingIndexedStep indexed();
 
 	/**
+	 * Define a type binder, responsible for creating a bridge.
+	 * To pass some parameters to the bridge,
+	 * use the method {@link #binder(TypeBinder, Map)} instead.
+	 *
+	 * @param binder A {@link TypeBinder} responsible for creating a bridge.
+	 * @return {@code this}, for method chaining.
+	 * @see TypeBinder
+	 */
+	default TypeMappingStep binder(TypeBinder binder) {
+		return binder( binder, Collections.emptyMap() );
+	}
+
+	/**
+	 * Define a type binder, responsible for creating a bridge.
+	 * With this method it is possible to pass a set of parameters to the binder,
+	 * so that they can be used by the bridge.
```

Comment:
```suggestion
	 * With this method it is possible to pass a set of parameters to the binder.
```

Parameters don't have to be used by the bridge, necessarily. They can just control mapping options (is this field sortable, ...).